org.cloudcompare.CloudCompare.yaml

```diff
@@ -57,7 +57,22 @@ cleanup:
 
     - /include
 
+    - /rdblib
+
 modules:
+  - name: rdblib
+    sources:
+      - type: archive
+        url: https://repository.riegl.com/software/libraries/rdblib/archive/2.3.7/linux/rdblib-2.3.7-x86_64-linux.tar.gz
+        sha256: 181ff40b6145c9af8ff0d38cfa2f89480bf57c308e63a810e0760a791cf5d75c
+    buildsystem: simple
+    build-commands:
+    - pwd
+    - ls -al .
+    - mkdir -p /app/rdblib
+    - cp -r . /app/rdblib
+    - ls /app/rdblib
```

Comment:
I only noticed now that this is a binary package. Is this also available in source? That would make it easier to also build on their architectures.

Reply:
The `pwd` and `ls` commands are not required to install the package. It's sufficient to create the directory and copy the files into it.